### DISCUSSION TITLE
如何正确使用通过<script>引入的js中的全局变量

### DISCUSSION THREAD

**Discussion by realhuhu** [vuejs/vitepress#4444] [Q&A]
通过head: [["script", {src: 'https://www.draw.io/js/viewer.min.js'}]]引入js，里面定义了一个全局变量GraphViewer，我需要在每篇md末尾调用一次GraphViewer.processElements()，在dev模式可以正常运行，但是build时会提示GraphViewer is not defined，即使declare const GraphViewer也一样，请问推荐做法是什么

**Comment by xycabcd** [comment]
推荐使用英文再问一遍

**Comment by charlesooo** [comment]
通过npm安装viewerjs，再在md里通过vue的方式调用
